src/debugger/scriptImporter.ts

```diff
@@ -86,4 +87,14 @@ export class ScriptImporter {
                 return new FileSystem().writeFile(sourceMappingLocalPath, updatedContent);
             });
     }
+
+    /**
+     * Changes the port of the url to be the one configured as this.packagerPort
+     */
+    private overridePackagerPort(urlToOverride: string): string {
```

Comment:
You could create a new class that handles all the multiple instance scenarios, and I would have the worker use it when calling. `ScriptImporter` should not care if we run one or multiple instances of VS Code.